R/estimate_marginal_models.R

```diff
@@ -90,6 +93,8 @@ estimate_marginal_models <- function(data,
           # extract the coefficients for the window
           coefs <- roll@model$coef[[window]]$coef[, 1]
           spec <- rugarch::ugarchspec(
+            mean.model = list(armaOrder = arma_order[2:3], include.mean = arma_order[1]),
```

Comment:
`include.mean` should be logical i.e. `TRUE `or `FALSE`. `arma_order[1]` is as far as I remember 1 so please adjust it to be `(arma_order[1] > 0)`. Of course the current approach works but let's keep the types clean.
![image](https://user-images.githubusercontent.com/46082939/207055137-a73f1829-d5bb-4b58-a802-0ccb2c901387.png)